guidelines/python-cli-ux-guidelines.md

```diff
@@ -0,0 +1,44 @@
+# Python CLI UX Guidelines
+
+1. The tool’s executable name should be “**conjur**” (regardless to its supported platform), so “conjur-py3-cli-darwin” is not a good experience.
+2. Once the CLI tool is executed **without** **any** **commands**, we should show the help and usage.
+3. Make sure to provide short-form options (e.g. -h) and long-form options (--help)
+4. Provide helpful and **readable** **feedback** (output) at all times. E.g., when a command succeeds and doesn’t have output, it is ended (in new line) with: Command succeeded! Data set to variable: XYZ
+5. Provide helpful and **readable** **errors** (we currently show exception and line code): E.g.,:  conjur exception cli_err01: command failed. No such file or directory: ‘/Users/sharonr/.conjurrc’
+6. When a command is executed with **missing** **parts** (subcommand, argument), or the CLI is executed with unfamiliar syntax, show error and right after the help of the command:
+
+conjur exception cli_err02: command not found. 
```

Comment:
IMO we don't even need the conjur exception. 

Just

ERROR: CLI001 Command not found

or

ERROR: CLI002 Missing parameter: VARIABLE

Reply:
BTW I don't even think we need a log number because it is just being printed to the user on the console

Reply:
I agree with removing "conjur exception" and using ERROR.
The log number is only for a troubleshooting reference, so the user would search for it in the docs and will get more info on how to solve the problem.